README.md

```diff
@@ -305,6 +305,56 @@ in your system configuration and
 
 in your Home Manager configuration.
 
+Flakes
+------
+
+Home Manager includes a flake.nix for compatibility with nixos flakes that wish
+to use it as a nixos module. An crude example of such usage follows:
+
+```nix
+{
+  description = "NixOS configuration";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+    home-manager.url = "github:rycee/home-manager";
+  };
+
+  outputs = inputs: {
```

Comment:
Thanks a lot. I also found the following snippet useful:

https://github.com/Mic92/dotfiles/blob/master/nixpkgs-config/homes.nix#L5

Here is how I use it to build home-manager in CI:

https://github.com/Mic92/dotfiles/blob/master/flake.nix#L85

Do you think it is worth adding as well?

Reply:
If @bqv can come up with a decent example, I think it would definitely be useful to see how to use the provided `lib` function -- especially since I haven't seen many people using it (you were actually the first).

Reply:
( It's actually @kisik21 who created and uses it, I don't :P )

Reply:
So, I'm not massively confident on the usage of this function, and I think it might be impolite to link directly to your config as an example of it :sweat_smile: I'll leave this for now, as at least those who wish to use it might want to use it have the direct use as a fallback, and this can still be amended later

Reply:
Ok. I might add an example later. It works great for me.